### PR TITLE
docs: Add deployed tags for local execution

### PR DESCRIPTION
**Pull request by jonatanschroeder**
As discussed on Slack, includes the tags for `us-prod-live` and `ca-live` for the latest deployed version in Docker Hub. `us-canary-live` is not added to avoid confusion.

### CODE REVIEW

**Comment by jonatanschroeder** [comment]
I also removed references to older numbered-versions, since those haven't been in use for a while.

File: docs/installing.md [hunk 1]
```diff
@@ -52,18 +52,22 @@ docker pull prairielearn/prairielearn
...
 
-To run a specific older version (e.g., version 1.2.3) then you can do:
+Make sure to also upgrade the Docker version periodically. Updates in the production servers are announced in the [PrairieLearn GitHub Discussions page](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/announcements). When an announcement is made, you are encouraged to run:
 
 ```sh
-docker run -it --rm -p 3000:3000 [other args] prairielearn/prairielearn:1.2.3
+docker pull prairielearn/prairielearn:us-prod-live
 ```
 
+Additional tags are available for older versions. The list of available versions is viewable on the [hub.docker build page](https://hub.docker.com/r/prairielearn/prairielearn/builds/).
```
Comment (nwalters512) [inline docs/installing.md:69]:
```suggestion
Additional tags are available for older versions. The list of available versions is viewable on the [Docker Hub build page](https://hub.docker.com/r/prairielearn/prairielearn/builds/).
```

File: docs/installing.md [hunk 2]
```diff
@@ -52,18 +52,22 @@ docker pull prairielearn/prairielearn
...
-The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version.
+The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version. If you would like to run the version that is currently deployed (or a specific older version), use a tag associated to the environment your course is running with. For PrairieLearn's main servers, the tags `us-prod-live` and `ca-live` match the deployed versions in the US and Canadian servers, respectively.
 
-The list of available versions is viewable on the [hub.docker build page](https://hub.docker.com/r/prairielearn/prairielearn/builds/).
+```sh
+docker run -it --rm -p 3000:3000 [other args] prairielearn/prairielearn:us-prod-live
+```
 
-To run a specific older version (e.g., version 1.2.3) then you can do:
+Make sure to also upgrade the Docker version periodically. Updates in the production servers are announced in the [PrairieLearn GitHub Discussions page](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/announcements). When an announcement is made, you are encouraged to run:
```
Comment (nwalters512) [inline docs/installing.md:63]:
```suggestion
Make sure to periodically pull new versions of the image. Updates in the production servers are announced in the [PrairieLearn GitHub Discussions page](https://github.com/PrairieLearn/PrairieLearn/discussions/categories/announcements). When an announcement is made, you are encouraged to run:
```

Alternatively, we could update the above command to use [`--pull=always`](https://docs.docker.com/engine/reference/commandline/run/#-set-the-pull-policy---pull)?

File: docs/installing.md [hunk 3]
```diff
@@ -52,18 +52,22 @@ docker pull prairielearn/prairielearn
 
 After this, run PrairieLearn using the same commands as above.
 
-## Running a specific older version of PrairieLearn
+## Running a specific version of PrairieLearn
 
-The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version.
+The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version. If you would like to run the version that is currently deployed (or a specific older version), use a tag associated to the environment your course is running with. For PrairieLearn's main servers, the tags `us-prod-live` and `ca-live` match the deployed versions in the US and Canadian servers, respectively.
```
Comment (nwalters512) [inline docs/installing.md:57]:
```suggestion
The commands above will always run the very latest version of PrairieLearn, which might be an unreleased development version. If you would like to run the version that is currently deployed (or a specific older version), use the appropriate tag for the server you're using:

- https://us.prairielearn.com: `us-prod-live`
- https://ca.prairielearn.com: `ca-live`
```

**Review by nwalters512 — APPROVED** [review]

File: docs/installing.md (hunk 3 above)
Comment (jonatanschroeder) [reply docs/installing.md]: I massaged the content above and added a note about other institutions in e172060b7.

File: docs/installing.md (hunk 2 above)
Comment (jonatanschroeder) [reply docs/installing.md]: I changed to `--pull=always` and replaced this statement with a request to restart instead of pull in 9bc2e03.

**Pull request merged**